content/en/blog/2023/announcing-notation-rc5.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Notation v1.0.0-RC.5 is available!
+author:  "Notation Release Team"
+date: 2023-05-18
+draft: false
+---
+
+The Notation maintainers are pleased to announce v1.0.0-RC.5 for [notation CLI](https://github.com/notaryproject/notation) and [notation-go library](https://github.com/notaryproject/notation-go). This blog walks you through the major updates in this release.
+
+## Major updates in notation and notation-go
+
+Notation is a CLI tool to sign and verify OCI artifacts. The v1.0.0-RC.5 changes include but are not limited to the following. See the [v1.0.0-RC.5 release notes](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.5) for details.
+
+* Improve the output message when there is no signature associated with the image being referred
```

Comment:
Image being referred in what CLI commands, we should give examples  "Notation list", "Notation inspect", Notation Verify.. any others?

Reply:
Added two commands